dymos/examples/hull_problem/test/test_hull_problem.py

```diff
@@ -94,3 +101,27 @@ def test_hull_shooting(self):
         assert_near_equal(p.get_val('traj.phase.timeseries.u')[-1],
                           uf,
                           tolerance=1e-4)
+
+    def test_hull_picard(self):
+        p = self.make_problem(transcription=PicardShooting(num_segments=3, nodes_per_seg=11),
+                              control_cnty=True,
+                              control_rate_cnty=True,
+                              control_rate2_cnty=False)
+        dm.run_problem(p, run_driver=True, simulate=True)
+
+        xf, uf = self.solution(1.5, 10)
+
+        assert_near_equal(p.get_val('traj.phase.timeseries.x')[-1],
+                          xf,
+                          tolerance=1e-4)
+
+        assert_near_equal(p.get_val('traj.phase.timeseries.u')[-1],
+                          uf,
+                          tolerance=1e-4)
+
+        assert_near_equal(p.get_val('traj.phase.continuity_comp.defect_controls:u'), np.zeros((2, 1)), tolerance=1.0E-4)
```

Comment:
Curious about the choice of 1e-4 for the tolerance. Is this related to the solver tols in the Picard method?

Reply:
No the test was always setup up for this tolerance.  Theres just a bit of noise in the optimal control profile and expecting more than 1E-4 is a bit of a stretch.